examples/jsm/renderers/webgpu/utils/WebGPUTextureUtils.js

```diff
@@ -156,6 +157,17 @@ class WebGPUTextureUtils {
 
 		}
 
+		if ( texture.isRenderTargetTexture && sampleCount > 1 ) {
+
+			const msaaTextureDescriptorGPU = Object.assign( {}, textureDescriptorGPU );
```

Comment:
I think `{ ...textureDescriptorGPU }` looks a bit clearer that it is just a clone.